### ISSUE TITLE
Proposed Keras RNN interface does not provide `sequence_length`

### ISSUE THREAD

**Issue by foxik** [tensorflow/community#36]
I am not sure if I am writing to a correct place, so please point me to a better place if this is not it.

According to the https://github.com/tensorflow/community/blob/master/rfcs/20180920-unify-rnn-interface.md RFC the TF RNN should become obsolete. However, it has a major important functionality -- the `sequence_length` parameter allowing to process sequences of different length in a single batch. For NLP, this is quite crucial. However, as far as I can see, the Keras RNN does not offer such functionality. Are there any plans to add it?

As it stands, many of our NLP models will not be expressable in TF 2.0 APIs.

**Comment by foxik** [comment]
I just came across [Ragged Tensors](https://www.tensorflow.org/guide/ragged_tensors). If the Keras RNN API supports them, that would solve the problem neatly. However, current `tf-nightly` does not and googling for planned support came out empty.

**Comment by foxik** [comment]
Adding @qlzh727 as the author of the Unify RNN Interface RFC.

**Comment by qlzh727** [comment]
Hi @foxik, the equivalent concept for sequence_length in Keras RNN layer is called "masking".  When your input data in the batch has different timestep length, instead of creating a sequence_length tensor in tf.rnn, in keras please add a keras.layers.Masking(), which will by default filter out 0 zeros in the input tensor.

**Comment by foxik** [comment]
Thanks! Sorry for taking your time, but I was unable to find it.
